cloud_functions/shared/logger_utils.py

```
@@ -37,3 +39,22 @@ def setup_logger(name: Optional[str] = None, level: int = logging.DEBUG) -> Logg
 
 
 logger = setup_logger()
+
+
+def log_structured(function_name: str, message: str, **kwargs):
```

Ruff (0.8.2)

Missing return type annotation for public function `log_structured`

Add return type annotation: `None`

(ANN201)